src/UI/Launcher.module.css

```diff
@@ -26,3 +26,9 @@
 	height: 35px;
 	padding: 10px 0 10px 0; /* Makes sure the img is centered correctly in the round button */
 }
+
+.launcher.state-open {
```

Comment:
launcher moet er volgends mij uit.

en hoort het leeg?

Reply:
Het hoort leeg, is een css class die we zetten om anderen css op te kunnen laten toevoegen.

Reply:
Nee hoort er in. Deze classes werden dynamisch opgebouwd en daarom kregen ze geen prefix. Via deze manier werkt het wel. Ze zijn leeg omdat wij geen styling ervoor bieden maar de klant gebruikt deze classes wel.